udata/core/reuse/models.py

```diff
@@ -23,6 +25,20 @@ class ReuseQuerySet(OwnedQuerySet):
     def visible(self):
         return self(private__ne=True, datasets__0__exists=True, deleted=None)
 
+    def visible_by_user(self, user: User) -> OwnedQuerySet:
+        """Return EVERYTHING visible to the user."""
+        public_qs: OwnedQuerySet = db.Q(private__ne=True, deleted=None, archived=None)
+        if user.is_anonymous:
+            return self(public_qs)
+
+        owners: list[User | Organization] = []
+        owned_qs: OwnedQuerySet = db.Q()
+        owners = list(user.organizations) + [user.id]
+        for owner in owners:
+            owned_qs |= db.Q(owner=owner) | db.Q(organization=owner)
```

Comment:
Couldn't we use [owned_by](https://github.com/opendatateam/udata/blob/c42e85883d5f796fe01ebb5acad8de979a4d040f/udata/core/owned.py#L21)?

Reply:
Same answer as above, we reuse it in `OwnedQuerySet.visible_by_user`